browser/app/js/actions.js

```diff
@@ -432,17 +432,23 @@ export const setLoginError = () => {
 
 export const downloadSelected = (url, req, xhr) => {
   return (dispatch) => {
+    var anchor = document.createElement('a')
+    document.body.appendChild(anchor);
     xhr.open('POST', url, true)
     xhr.responseType = 'blob'
 
     xhr.onload = function(e) {
       if (this.status == 200) {
         dispatch(checkedObjectsReset())
         var blob = new Blob([this.response], {
-          type: 'application/zip'
+          type: 'octet/stream'
         })
         var blobUrl = window.URL.createObjectURL(blob);
-        window.location = blobUrl
+        anchor.href = blobUrl
+        anchor.download = req.bucketName+'-'+req.prefix.slice(0, -1)+'.zip';
```

Comment:
what is the final name of the file downloaded ? can you provide an example say bucketName is "testbucket"  and prefix is "a/b/c/d/movies/"